### PR TITLE
Add support for  markdown in comment notification emails.

### PR DESCRIPTION
**Pull request by kch8qx**
Comment notification emails now support all markdown that is also supported in comments using markdown-it, including:

* Links
* Headers
* Horizontal lines
* Emphasis (bold, italic, underlined, strike-through)
* Lists (ordered and unordered)
* Code blocks
* Images
* Tables

Only Commonmark markdown unsupported by comment notification emails is the markdown also unsupported by comment markdown-it JS (blockquotes, emojis, superscript/subscript, marked text, etc.)

[OSF-5559]

### CODE REVIEW

**Comment by sloria** [comment]
@kch8qx Can you please look into the failing tests?

File: website/static/js/contribRemover.js [hunk 1]
```diff
@@ -267,4 +267,4 @@ ContribRemover.prototype.init = function() {
     });
 };
 
-module.exports = ContribRemover;
+module.exports = ContribRemover;
```
Comment (caseyrollins) [inline website/static/js/contribRemover.js]: Can you add a newline to the end of this file?

File: website/static/js/pages/base-page.js [hunk 2]
```diff
@@ -122,4 +122,4 @@ $(function() {
         }
 
     }
-});
+});
```
Comment (caseyrollins) [inline website/static/js/pages/base-page.js]: Newline at EOF.

**Comment by caseyrollins** [comment]
Looks good! :tada: Just needs the small EOF changes.

**Comment by kch8qx** [comment]
@caseyrollins Just added the new lines and committed!

**Comment by caseyrollins** [comment]
@kch8qx, can you update with develop to fix the build? After that, RTM :star:.

**Comment by kch8qx** [comment]
@caseyrollins sorry about that, should now be updated to latest develop and good to go!

**Comment by kch8qx** [comment]
@caseyrollins noticed earlier that there was a merge conflict with the latest develop, so this is now fixed!

**Comment by sloria** [comment]
Awesome; thanks!

**Pull request merged**